automatic/git.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,25 @@
+﻿$ErrorActionPreference = 'Stop'
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\helpers.ps1
+
+$filePath32  = "$toolsPath\Git-2.11.0-32-bit.exe"
+$filePath64  = "$toolsPath\Git-2.11.0-64-bit.exe"
+$installFile = if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true') {
+                      Write-Host "Installing 64 bit version"; $filePath64 }
+               else { Write-Host "Installing 32 bit version"; $filePath32 }
+
+$pp = Get-PackageParameters
+Set-InstallerRegistrySettings $pp
+
+Stop-GitSSHAgent
+
+$packageArgs = @{
+    PackageName    = 'git.install'
+    FileType       = 'exe'
+    SoftwareName   = 'Git version *'
+    File           = $installFile
+    SilentArgs     = "/VERYSILENT", "/SUPPRESSMSGBOXES", "/NORESTART", "/NOCANCEL", "/SP-", "/LOG", (Get-InstallComponents $pp)
+    ValidExitCodes = @(0)
+}
+Install-ChocolateyInstallPackage @packageArgs
+rm -Force $filePath32, $filePath64 -ea 0
```

Comment:
That was added by @ferventcoder and should probably get removed. It doesn't represent guideline if you have an option not to use it - use sparingly is open to interpretation. Other reasons mentioned are moot:

>They tend to make PowerShell harder to read, 

- It depend on alias (cross-platform or not?)

> Sometimes they are not available in all supported versions of PowerShell

This is redudant rule - we already have posh2 compatibility rule.

> and someone can override aliases locally 

The argument is invalid. Someone can override anything locally, cmdlets included. The aliases do not fix that but only Full Quoalified Cmdlet Name which then makes it unreadable. default aliases are read only so this is even incorrect.